recipe/bld.bat

```diff
@@ -3,13 +3,18 @@ setlocal EnableDelayedExpansion
 mkdir "%SRC_DIR%"\build
 pushd "%SRC_DIR%"\build
 
+if "%gcs%"=="gcs_enabled" (
```

Comment:
Does there need to be a `TILEDB_GCS=OFF` default here?

Reply:
Oh, hmm - good catch. CMake does default to "OFF" (just tested) but I can add it for completeness

Reply:
Updated